src/App.jsx

```diff
@@ -14,70 +15,60 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleGoodSelection = (good) => {
+    setSelectedGood(prevSelectedGood => (
+      prevSelectedGood === good ? '' : good));
+  };
```

Comment:
Interesting solution. Also you can create separate function for clear good
```suggestion
  const handleGoodSelection = (good) => {
    setSelectedGood(prevSelectedGood => (
      prevSelectedGood === good ? '' : good
    ));
  };
```